scout/adapter/mongo/managed_variant.py

```diff
@@ -113,36 +114,36 @@ def find_managed_variant_id(self, variant_id):
         """Fetch eg search for a managed variant with the encoded positional variant_id.
 
         Arguments:
-            display_id(str): chrom_pos_ref_alt_category_build
+            chromosome, position, reference, alternative, "clinical"
+            variant_id(str): md5(chrom_pos_ref_alt_category_type)
                 category: "snv", "cancer_snv" - "sv", "cancer_sv" possible but not expected
-                build: "37" or "38"
+                type: "clinical"
```

Comment:
Perhaps add types also on line 113?

Reply:
Not sure I follow: that is a for-quick-reference expansion of the variant_id md5 string stuff, not a formal argument. It might be neat to have a strict type for it I guess. I can add str att least! 😁